LillyJudgeHW1.md

```diff
@@ -0,0 +1,63 @@
+---
+title: A Short Biography of Lilly Judge
+author: "Lilly J"
+date: "10/6/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Profile
+
+ ![Lilly circa 2018 at a winery in Virginia](/Users/lgjudge/Desktop/e1GnJs7__400x400.jpg)
```

Comment:
This is referring to a location on your computer -- GitHub can't access that location on the internet.